src/quicklist.h

```diff
@@ -200,6 +199,8 @@ int quicklistBookmarkCreate(quicklist **ql_ref, const char *name, quicklistNode
 int quicklistBookmarkDelete(quicklist *ql, const char *name);
 quicklistNode *quicklistBookmarkFind(quicklist *ql, const char *name);
 void quicklistBookmarksClear(quicklist *ql);
+int quicklistisLargeElement(size_t size);
```

Comment:
i don't think this method should be declared in the header file, i think it can be a static one in the c file, or maybe even a macro or not needed at all.
the other one is needed here only because of the debug command (hack)